doc/api/configuration.md

```diff
@@ -241,3 +241,33 @@ print(res)
 res = %sql SELECT * FROM languages LIMIT 2
 print(res)
 ```
+
+## `persist_snippets`
+
+```{versionadded} 0.8.1
+```
+
+Default: `False`
+
+Stores the snippet in a SQL file.
+
+```{code-cell} ipython3
+%config SqlMagic.persist_snippets = True
+```
+
+```{code-cell} ipython3
+%%sql --save rated_high
+select * from languages where rating > 10.5
+```
```

Comment:
I noticed that if I created a snippet and then a second snippet that depends on the first one, the content stored for the first one does not include the `WITH` statement, but it should. essentially, the SQL file should contain the output of `%sqlcmd snippets {name}`

notebook: [testing-persist-sql.zip](https://github.com/ploomber/jupysql/files/12186654/testing-persist-sql.zip)

Reply:
Resolved this issue, now when we store the contents of a dependent snippet, it includes the `WITH` clause